src/api/app/services/trigger_controller_service/scm_extractor.rb

```diff
@@ -16,19 +18,30 @@ class SCMExtractor
       }
     }.freeze
 
+    # FIXME: This is a factory
     def initialize(scm, event, webhook_payload)
-      # TODO: What should we do when the user sends a wwwurlencoded payload? Raise an exception?
-      @webhook_payload = webhook_payload.deep_symbolize_keys
       @scm = scm
       @event = event
+      # TODO: What should we do when the user sends a wwwurlencoded payload? Raise an exception?
```

Comment:
Would transform this TODO into a card in the backlog to deal with it and remove the comment, its a bit lost here IMO

Reply:
[Done](https://trello.com/c/FFw9eU5r/244-what-should-we-do-when-the-user-sends-a-wwwurlencoded-payload-raise-an-exception)